presto-main/src/main/java/com/facebook/presto/operator/scalar/StringFunctions.java

```diff
@@ -617,6 +617,35 @@ public static Slice rightPad(@SqlType("varchar(x)") Slice text, @SqlType(Standar
         return pad(text, targetLength, padString, text.length());
     }
 
+    @Description("computes Levenshtein Distance between two strings")
+    @ScalarFunction
+    @LiteralParameters({"x", "y"})
+    @SqlType(StandardTypes.BIGINT)
+    public static long levenshteinDist(@SqlType("varchar(x)") Slice string1, @SqlType("varchar(y)") Slice string2)
```

Comment:
Make the functions take code points as arguments, to avoid conversion for constants. See `leftTrim` as an example.